### PR TITLE
update placement doc

### PR DESCRIPTION
**Pull request by haoqing0110**
Ref: https://github.com/open-cluster-management/backlog/issues/16066
Signed-off-by: haoqing0110 <qhao@redhat.com>

### CODE REVIEW

**Comment by haoqing0110** [comment]
@cadawson could you help review this?

File: clusters/placement_managed.adoc [hunk 1]
```diff
@@ -37,6 +37,19 @@ If not specified, all `ManagedClusters` that meet the placement requirements are
 
 - `Predicates` represents a slice of predicates to select `ManagedClusters` with label and claim selector. The predicates are ORed.
 
+- `prioritizerPolicy` represents the policy of prioritizers. 
+  - `mode` is either `Exact`, `Additive`, `""` where `""` is Additive by default.
```
Comment (cadawson) [inline clusters/placement_managed.adoc]: Why do we have `""` if it is `Additive` by default?

**Review by cadawson — COMMENTED** [review]
Looks good! I am trying to figure out more about placements. I invited you to view a slide. I also added a PR for this review. Thanks!

**Comment by cadawson** [comment]
Here are the changes, in case the PR is gone: 
[placements_update.txt](https://github.com/open-cluster-management/rhacm-docs/files/7413717/placements_update.txt)

**Review by cadawson — APPROVED** [review]
Looks good! Thank you!

**Pull request merged**